include/ranges.h

```diff
@@ -422,7 +429,7 @@ namespace detail {
 	  public:
 		subscript_proxy(Target& tgt, const id<TargetDims> id) : m_tgt(tgt), m_id(id) {}
 
-		inline decltype(auto) operator[](const size_t index) const {
+		inline decltype(auto) operator[](const size_t index) const { //
```

Comment:
I updated my clangd, and it decided to format this differently. Since at some point we'll probably want to update, I went with this, since it causes both the CI version and my current (more up to date) version to format it the same.